phpcs.xml

```diff
@@ -9,6 +9,12 @@
     <exclude-pattern>*/TestAsset/*</exclude-pattern>
     <exclude-pattern>*/_files/*</exclude-pattern>
 
+    <rule ref="Generic.Files.LineLength.TooLong">
```

Comment:
@Ocramius @weierophinney I added this ignore rule because the annotations for psalm I added were quite long.

Reply:
You can split annotations over multiple lines, especially those detailing array or object structures; I tend to prefer that approach as it has the side effect of making them more readable.

Reply:
That worked indeed. I pushed the change.